src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -290,7 +290,7 @@ private static FormulaValue RunAggregator(IAggregator agg, EvalVisitor runner, S
                 if (row.IsValue)
                 {
                     var childContext = context.WithScopeValues(row.Value);
-                    var value = arg1.EvalAsync(runner, childContext).Result;
```

Comment:
>Result [](http://example.com/codeflow?start=69&length=6)

This was a real bug, caught by the new AsyncVerify tests.

Reply:
awesome!

Reply:
@LucGenetier  - here's an example of a hang.